website/docs/docs/fusion/fusion-readiness.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Fusion readiness checklist
+id: fusion-readiness
+description: "Your to-do list for preparing for Fusion"
+sidebar_label: "Fusion readiness checklist"
+pagination_next: null
+pagination_prev: null
+---
+
+The <Constant name="fusion_engine" /> is here! We currently offer it as a [private preview](/docs/dbt-versions/product-lifecycles#the-dbt-platform) on the <Constant name="dbt_platform" />. Even if we haven't enabled it for your account, you can still take steps to prepare your projects for upgrade. If this is all new to you, first [read about <Constant name="fusion" />](/docs/fusion), what it is, its current state, and the features available. Then reference this checklist to begin preparing your projects for <Constant name="fusion" />.
+
+## Preparing for Fusion
+
+1. [ ] **Upgrade to the most recent version of dbt:** Make sure all your projects are on the `Latest` [release track](/docs/dbt-versions/cloud-release-tracks) across all deployment environments and jobs. This will ensure the simplest, most predictable experience by allowing you to pre-validate that your project doesn't rely on deprecated behaviors. 
```

Comment:
```suggestion
1. [ ] **Upgrade to the latest dbt version:** Make sure all your projects are on the `Latest` [release track](/docs/dbt-versions/cloud-release-tracks) across all deployment environments and jobs. This will ensure the simplest, most predictable experience by allowing you to pre-validate that your project doesn't rely on deprecated behaviors. 
```

Reply:
Alternatively, you could make the 1st level steps headers:
> ## 1. Upgrade to the latest dbt version